src/App.js

```diff
@@ -36,23 +48,73 @@ class App extends Component {
           }})
   }
 
-  render() {
-    const {movies, isLoading, errorMsg} = this.state;
+  goHome() {
+    this.setState({ atHome: true })
+  }
 
-    if(errorMsg) {
-      return <p className='error-message'>{errorMsg}</p>
+  leaveHome() {
+    this.setState({ atHome: false })
+  }
+
+  logIn(user) {
+      this.setState({ user: user })
     }
 
-    if(isLoading) {
-      return <p className='loading-message'>Loading...</p>
+  logOut() {
+      this.setState({ user: null })
     }
 
+
+  render() {
+    const {movies, isLoading, errorMsg, atHome, user} = this.state;
+
     return (
       <>
-        <main>
-          <TopRated movies={movies}/>
-          <Movies movies={movies}/>
-        </main>
+      <header>
+        <nav className="header-content">
+          <h1>
+            <NavLink to={{
+              pathname:'/'
+            }}  className="site-title" onClick={this.state.goHome}>Rancid<br/> Tomatillos
+            </NavLink>
+          </h1>
+            <NavLink to={{
+              pathname:'/'
+              }}>
+              {!user && <button>Log in</button>}
+              {user && <button>Log out</button>}
+            </NavLink>
+          {!atHome &&
+            <NavLink to={{
+              pathname:'/'
+              }}>
+              <img src={homeButton} alt="home button" className='home-button' onClick={this.state.goHome}/>
+            </NavLink>
+          }
+        </nav>
+        <section className="gradient"></section>
+      </header>
+        <Switch>
+          <Route path='/' exact render={() => <Home
+            errorMsg={errorMsg}
+            isLoading={isLoading}
+            movies={movies}
+            leaveHome={this.state.leaveHome}
+            />} />
+          <Route path='/movie-details/:title' exact component={MovieDetails} />
+          <Route path='/about' exact component={About} />
+          <Route path='/faq' exact component={FAQ} />
+          <Route path='/contact-us' exact component={ContactUs} />
+          <Route path='/' render={() => <div>404</div>} />
+        </Switch>
+        <footer>
+          <section className="gradient"></section>
+          <nav className="footer-links">
+            <NavLink className="about footer-link" to='/about' onClick={this.state.leaveHome}>About</NavLink>
+            <NavLink className="faq footer-link" to='/faq' onClick={this.state.leaveHome}>FAQ</NavLink>
+            <NavLink className="contact-us footer-link" to='/contact-us' onClick={this.state.leaveHome}>Contact Us</NavLink>
```

Comment:
Nice job on changing the header and footer to be NavLinks instead of just Links! Gives more practice with other properties of React Router